### ISSUE TITLE
Some icons not high-DPI ready

### ISSUE THREAD

**Issue by ofadam** [jks-prv/Beagle_SDR_GPS#238]
Some of the buttons are not high-DPI, making them appear pixelated on screens like Apple's MacBook Pro line. Could they be changed to 2x references in the template to make them smooth?

<img width="355" alt="Screen Shot 2019-05-07 at 9 15 08 PM" src="https://user-images.githubusercontent.com/5504940/57344529-389f7a80-710d-11e9-98d2-c972d710a2d5.png">

**Comment by jks-prv** [comment]
Yeah these icons were done a long time ago by hand before I started to use Font Awesome for everything.

**Comment by ofadam** [comment]
I believe that if you make sure that the source images are 2x the normally required resolution, but then reference it at the required resolution in the code, it should make it high-DPI ready.

**Comment by ofadam** [comment]
I've adjusted some of the code in a recent pull request. I'll try to continue the work when able.
